### PR TITLE
PR 3 - Implement toolkit csharp samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
**Toolkit Migration Sample -**
     1. bot-adaptive-card-actions/csharp
     2. bot-conversation/csharp
     3. bot-daily-task-reminder/csharp
     4. tab-meeting-transcript-recording/csharp
     5. tab-request-approval/csharp
     6. tab-sso/csharp

### CODE REVIEW

File: samples/bot-adaptive-card-actions/csharp/README.md [hunk 1]
```diff
@@ -37,6 +37,19 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 - [dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) or [ngrok](https://ngrok.com/) latest version or equivalent tunnelling solution
 
 - [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+- [Teams Toolkit for Visual Studio](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/toolkit-v4/install-teams-toolkit-vs?pivots=visual-studio-v17-7)
+
+## Run the app (Using Teams Toolkit for Visual Studio)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio.
+1. Install Visual Studio 2022 **Version 17.9 or higher** [Visual Studio](https://visualstudio.microsoft.com/downloads/)
```
Comment (formulahendry) [inline samples/bot-adaptive-card-actions/csharp/README.md:45]: For all the bot projects, need to be `Visual Studio 2022 version 17.9 Preview 2 or higher`

File: samples/bot-daily-task-reminder/csharp/README.md [hunk 2]
```diff
@@ -44,6 +44,20 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 
 - [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
 
+- [Teams Toolkit for Visual Studio](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/toolkit-v4/install-teams-toolkit-vs?pivots=visual-studio-v17-7)
+
+## Run the app (Using Teams Toolkit for Visual Studio)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio.
+1. Install Visual Studio 2022 **Version 17.9 or higher** [Visual Studio](https://visualstudio.microsoft.com/downloads/)
```
Comment (formulahendry) [inline samples/bot-daily-task-reminder/csharp/README.md:52]: For all the bot projects, need to be `Visual Studio 2022 version 17.9 Preview 2 or higher`

File: samples/bot-conversation/csharp/README.md [hunk 3]
```diff
@@ -39,6 +39,19 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 - Microsoft Teams is installed and you have an account
 - [.NET SDK](https://dotnet.microsoft.com/download) version 6.0
 - [dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) or [ngrok](https://ngrok.com/) latest version or equivalent tunnelling solution
+- [Teams Toolkit for Visual Studio](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/toolkit-v4/install-teams-toolkit-vs?pivots=visual-studio-v17-7)
+
+## Run the app (Using Teams Toolkit for Visual Studio)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio.
+1. Install Visual Studio 2022 **Version 17.9 or higher** [Visual Studio](https://visualstudio.microsoft.com/downloads/)
```
Comment (formulahendry) [inline samples/bot-conversation/csharp/README.md:47]: For all the bot projects, need to be `Visual Studio 2022 version 17.9 Preview 2 or higher`

File: samples/bot-adaptive-card-actions/csharp/README.md (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/bot-adaptive-card-actions/csharp/README.md:45]: Fixed

File: samples/bot-daily-task-reminder/csharp/README.md (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/bot-daily-task-reminder/csharp/README.md:52]: Fixed

File: samples/bot-conversation/csharp/README.md (hunk 3 above)
Comment (Harikrishnan-MSFT) [reply samples/bot-conversation/csharp/README.md:47]: Fixed

**Review by formulahendry — APPROVED** [review]

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, approving!

**Comment by Harikrishnan-MSFT** [comment]
**_Build-Pipeline-
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/7955186657_**

**Pull request merged**